website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -151,6 +152,31 @@ A run will be triggered in this workspace when changes are detected in any of th
 }
 ```
 
+A run will be triggered in this workspace when Terraform Cloud detects any of the following changes:
```

Comment:
On [line 130](https://github.com/hashicorp/terraform-docs-common/blame/6b51020614afcccabde9d2975eea657604711c78/website/docs/cloud-docs/api-docs/workspaces.mdx#L130), it says "for a monorepo". I think we should add documentation here to state "when using prefixes, ..." and "when using patterns, ..."

Reply:
wait, I'm confused by what you mean by "in a bolded subsection". OH do you mean we need to set this off with a header the way "with monorepo" is?

GOOD CATCH.  Let me add a suggestion to see if that's what you mean.

Reply:
Yup, this looks good!